misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/SpecBodyExtensions.kt

```diff
@@ -24,3 +26,6 @@ fun SpecBody.prefixedDescribe(prefix: String, description: String, body: SpecBod
 
 fun SpecBody.prefixedDescribe(prefix: String, suffix: String, description: String, body: SpecBody.() -> Unit) =
     group("${prefix}describe$suffix $description", body = body)
+
+//Tried copying Spek 1 wrap function
+fun Root.include(root: Root.() -> Unit) = root(this)
```

Comment:
td;lr please remove

I had added this functionality myself but pushed it back to spek in the meantime. Moreover #193 removed exactly this function and rewrote the code so that we are using spek's include.